### ISSUE TITLE
Feature Request: Hint on Cards for changes or new entrys

### ISSUE THREAD

**Issue by Lacour20** [wekan/wekan#2111]
Hello,

first of all sorry for my bad english. Is it possible to add a feature that cards with new / unread changes are marked with a hint / notification symbol? Maybe a alarm clock or something else smiliar to trello?

Best Regards

Jan

**Comment by xet7** [comment]
@Lacour20 

Wekan does not yet save data about all changes, so that needs to be added first as optional feature, because it grows size of database much faster.

**Comment by Lacour20** [comment]
@xet7 oh ok. maybe you can bind it with the changes which are displayed in the activity bar. in my opinion the most importend changes are displayed there.

**Comment by xet7** [comment]
@Lacour20 

What should it look like? Please add screenshot mockup, so I can think what is possible.

**Comment by xet7** [comment]
@Lacour20 

Screenshot of Trello changes or screenshot mockup of wekan, as comment to this issue, with image attachment.

**Comment by Lacour20** [comment]
Sure. Here you see a card where two changes are made. i dont need a counter or something like this. a simple colored hint would be perfect

![hint](https://user-images.githubusercontent.com/44357002/51321344-ce31d400-1a62-11e9-8e49-1f985eeb98a3.PNG)

**Comment by xet7** [comment]
Hmm, I'm trying to think while drinking coffee. Maybe:

1) When something like card change is added to activities

2) Colored hint is added to card for each board user

3)  When each user clicks on card, colored hint is removed

And maybe this setting would be

a) user-specific setting to enable

b) board-specific setting to enable

c) Admin Panel global setting to enable

What do you think?

**Comment by xet7** [comment]
Maybe this could be like user-specific subscriptions, like there is for getting changes in email?

So each user could select what notifications they want to subscribe with email / browser notification / hint on card / etc? And user would select which activities to subscribe to and how?

**Comment by xet7** [comment]
@saurabharch 

What do you think? I think this is related to #2026

**Comment by Lacour20** [comment]
> So each user could select what notifications they want to subscribe with email / browser notification / hint >on card / etc? And user would select which activities to subscribe to and how?

for me it would be enough to say: "Acitvate Subscriptions on Cards"   .

**Comment by saurabharch** [comment]
@xet7 yeah similar but it is on page it didn’t trigger native push notifications because if any changes countinously it sounds like spam. We just send priority based notification message not all event.

**Comment by saurabharch** [comment]
@xet7 yeah it is also a good suggestion. And I like this thank you @lacour20 for your suggestion

**Comment by Lacour20** [comment]
Thanks to you guys. Wekan is a wonderful tool and you're doing a great job!

Regards
